### DISCUSSION TITLE
WEBDAV同步设置功能

### DISCUSSION THREAD

**Discussion by chenyu-01** [the1812/Bilibili-Evolved#1749] [功能建议 / Ideas]
# webdav同步需求
很多网盘已经支持了webdav，数据同步到网络可以做到实时更新。

**Comment by FoundTheWOUT** [comment]
尝试过，很遗憾的是作为一个油猴脚本，跨域请求只支持GET，POST，HEAD（至少我看文档只有这三个），然后要进行类似于webdav的同步可能还需要其它请求方法，比如PUT。
